### PR TITLE
[#91068930] increased version number, added endpoint examples

### PR DESCRIPTION
**Pull request by slamice**
- Added examples for jobs-put endpoint.
- Added to API endpoint map.

related to pr in [api-docs]()

### CODE REVIEW

File: examples/updateTranslationJobs.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                 'captcha': 'BBBB'}
+                ],
+})
+
+# Approve a group of jobs
+gengo.updateTranslationJobs(action={
+  'action': 'approve',
+  'job_ids': [{'job_id': 1077, 'comment': 'Keep up the good work'},
+              {'job_id': 629, 'comment': 'Great job!'}],
+})
```
Comment (hayate) [inline examples/updateTranslationJobs.py]: add new line please

File: examples/updateTranslationJobs.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# All code provided from the http://gengo.com site, such as API example code
+# and libraries, is provided under the New BSD license unless otherwise
+# noted. Details are below.
+#
+# New BSD License
+# Copyright (c) 2009-2012, myGengo, Inc.
```
Comment (hayate) [inline examples/updateTranslationJobs.py]: which Gengo? your Gengo? or my Gengo?

File: examples/updateTranslationJobs.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are
+# met:
+#
+# Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+# Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+# Neither the name of myGengo, Inc. nor the names of its contributors may
```
Comment (hayate) [inline examples/updateTranslationJobs.py]: ditto

Comment (kelvintaywl) [inline examples/updateTranslationJobs.py]: do we need to update the year to `2009-2015`?

**Comment by hayate** [comment]
LGTM

**Comment by kelvintaywl** [comment]
LGTM :hamburger:

**Pull request merged**